prow/plugins/trigger/ic.go

```diff
@@ -75,7 +76,7 @@ func handleIC(c client, ic github.IssueCommentEvent) error {
 			return err
 		}
 		if !trusted {
-			resp := fmt.Sprintf("you can't request testing unless you are a [%s](https://github.com/orgs/%s/people) member", trustedOrg, trustedOrg)
+			resp := fmt.Sprintf("you can't request testing unless you are a [%s](https://github.com/orgs/%s/people) member", strings.Join(c.GitHubClient.GetTrustedOrgs(), ","), strings.Join(c.GitHubClient.GetTrustedOrgs(), ","))
```

Comment:
changed messaging to: "you can't request testing unless you are a member of one of these orgs: %s"